docs/topics/unsigned-integer-types.md

```diff
@@ -2,16 +2,22 @@
 
 In addition to [integer types](numbers.md#integer-types), Kotlin provides the following types for unsigned integer numbers:
 
-* `UByte`: an unsigned 8-bit integer, ranges from 0 to 255
-* `UShort`: an unsigned 16-bit integer, ranges from 0 to 65535
-* `UInt`: an unsigned 32-bit integer, ranges from 0 to 2^32 - 1
-* `ULong`: an unsigned 64-bit integer, ranges from 0 to 2^64 - 1
+| Type     | Size (bits) | Min value | Max value                                       |
+|----------|-------------|-----------|-------------------------------------------------|
+| `UByte`  | 8           | 0         | 255                                             |
+| `UShort` | 16          | 0         | 65,535                                          |
+| `UInt`   | 32          | 0         | 4,294,967,295 (2<sup>32</sup> - 1)              |
+| `ULong`  | 64          | 0         | 18,446,744,073,709,551,615 (2<sup>64</sup> - 1) |
+
 
 Unsigned types support most of the operations of their signed counterparts.
 
 > Unsigned numbers are implemented as [inline classes](inline-classes.md) with the single storage property of the corresponding 
-> signed counterpart type of the same width. Nevertheless, changing type from unsigned type to signed counterpart (and vice versa) 
-> is a _binary incompatible_ change.
+> signed counterpart type of the same width. However, due to the compiler's mangling of signatures containing unsigned types, 
+> changing the type from unsigned to its signed counterpart or the other way around 
+> is a _binary incompatible_ change. 
+> 
+> For more information, see [Mangling](inline-classes.md#mangling).
```

Comment:
It's not clear to me how mangling makes conversion between signed and unsigned integer types binary incompatible. Can you explain it a bit more?

Also, it reads like mangling only occurs for unsigned types. Is this correct?